src/styles/post-template.less

```diff
@@ -341,7 +363,7 @@ body {
     .body {
         margin-top: 30px;
         line-height: 1.5;
-        width: 500px;
+        width: 580px;
```

Comment:
Can you speak more about why this is necessary?  I know the original author made the desktop page narrow on purpose -- though I don't know what that purpose was exactly -- so I'm hesitant to increase the width.  How does it affect the layout if you leave this at 500px?

Reply:
Sure thing! I assume the container width was set this way in order to maintain a readable line length (i.e. characters/words that fit in the column). I increased the width of the container in order to keep the line length roughly the same with the increased font size. A positive consequence of this is that there is also less unused whitespace (as Josh points out in #68) .

Reply:
@brownhead can you speak to why you made the width what you did?  Does this change seem appropriate to you?

Reply:
Ah sorry, my notifications are a bit of a mess on github. Was following far too many people.

@sloria is right, the width was mainly a function of the line length, so increasing it makes sense to me. I will posthumously pile on my support for this 👍